actionview/lib/action_view/helpers/tags/check_box.rb

```diff
@@ -57,7 +57,7 @@ def checked?(value)
           end
 
           def hidden_field_for_checkbox(options)
-            @unchecked_value ? tag("input", options.slice("name", "disabled", "form").merge!("type" => "hidden", "value" => @unchecked_value)) : "".html_safe
+            @unchecked_value.to_s.present? ? tag("input", options.slice("name", "disabled", "form").merge!("type" => "hidden", "value" => @unchecked_value)) : "".html_safe
```

Comment:
`present?` here will make `""` not be a valid unchecked value anymore. I think we should just do
```suggestion
            !@unchecked_value.nil? ? tag("input", options.slice("name", "disabled", "form").merge!("type" => "hidden", "value" => @unchecked_value)) : "".html_safe
```

A test is needed for the `""` as unchecked value.

Reply:
Thanks, @rafaelfranca ! Will do